src/services/requestService.js

```diff
@@ -95,26 +124,7 @@ export default class RequestService {
   static async availOpenRequests({ user: { id } }) {
     const openRequests = await Request.findAll({
       where: { status: 'open' },
-      include: [{
```

Comment:
Nice job make this block of code reusable

Reply:
Thanks